src/screens/ProfileScreen.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { View, Text, Button } from 'react-native';
+
+export default class Profile extends Component {
+  render() {
+    return (
+      <View>
+        <Text>Profile Screen</Text>
+        <Button title="go home" onPress={() => this.props.navigation.navigate('Home')} />
```

Comment:
This component does not have props validation. It is recommended to validate your props.

Reply:
Yes. I did not bother to add props validation because this is a demo page to illustrate the navigation. I have added the validation now, nonetheless.